compiler/semgnrc.nim

```diff
@@ -74,7 +74,7 @@ proc semGenericStmtSymbol(c: PContext, n: PNode, s: PSym,
   of skProc, skFunc, skMethod, skIterator, skConverter, skModule:
     result = symChoice(c, n, s, scOpen)
   of skTemplate:
-    if macroToExpandSym(s):
+    if sfCustomPragma notin s.flags and macroToExpandSym(s):
```

Comment:
Shouldn't this check be moved into ``macroToExpandSym``?

Reply:
Done, I wasn't 100% sure about the other callsite (`skMacro`) but it should be safe